embulk-output-sqlserver/src/main/java/org/embulk/output/sqlserver/SQLServerOutputConnector.java

```diff
@@ -1,26 +1,23 @@
 package org.embulk.output.sqlserver;
 
+import org.embulk.output.jdbc.JdbcOutputConnector;
+
 import java.sql.Connection;
-import java.sql.DriverManager;
+import java.sql.Driver;
 import java.sql.SQLException;
 import java.util.Properties;
 
-import org.embulk.output.jdbc.JdbcOutputConnector;
-
 public class SQLServerOutputConnector
         implements JdbcOutputConnector
 {
     private final String url;
     private final Properties properties;
     private final String schemaName;
+    private final Driver driver;
 
-    public SQLServerOutputConnector(String url, Properties properties, String schemaName)
+    public SQLServerOutputConnector(Driver driver, String url, Properties properties, String schemaName)
```

Comment:
We don't need `Driver` instance because `DriverManager#getConnection` will use appropriate `Driver` instance depending on the url.